act-rules-format.md

```diff
@@ -497,6 +523,65 @@ Background (optional) {#background}
 An ACT Rule <em class="rfc2119">may</em> contain information about the background for the development of the rule, or references to relevant reading. The background information is optional, but whenever it is included in the rule, the relationship to the relevant reading can be specified. Examples of relevant background references for a rule for a [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) [[WCAG]] success criterion could be [WCAG 2.1 Understanding documents](https://www.w3.org/WAI/WCAG21/Understanding/), [WCAG 2.1 Techniques](https://www.w3.org/WAI/WCAG21/Techniques/), or [WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria/), CSS [[css-2018]], or HTML [[HTML]] specifications.
 
 
+Implementations (optional) {#implementation}
+--------------------------------
+
+An ACT Rule <em class="rfc2119">may</em> contain a list of implementations that are consistent with that rule. An [=implementation=] is an accessibility test methodology or test tool that can be used to produce [=outcomes=] of an ACT Rule.
+
+An [=implementation=] consists of multiple [=implementation procedures=]. The [=Implementation procedures=] are not required to have a one-to-one mapping to an ACT Rule. Mapping of an implementation to an ACT Rule is done by determining the set of implementation procedures with the highest consistency and coverage of that rule. Multiple implementation procedures can combine to map to a single ACT Rule, or a single implementation procedure can map to multiple ACT rules simultaneously.
+
+For each implementation the following data <em class="rfc2119">may</em> be provided. If included, consistency and coverage <em class="rfc2119">must</em> be determined as defined in this section.
+
+- Name of implementation
+- Name of the vendor
+- [Consistency](#impl-consistency)
+- [Coverage](#impl-coverage) 
+- Version used in testing consistency and coverage
```

Comment:
Since this is a "may be provided" list, should we add mode (auto/semi/manual) to it?

Reply:
Sure thing!